oidc-service-impl/src/main/java/io/mosip/esignet/services/CacheUtilService.java

```diff
@@ -141,4 +153,13 @@ public OIDCTransaction getLinkedSessionTransaction(String linkTransactionId) {
     public OIDCTransaction getLinkedAuthTransaction(String linkTransactionId) {
         return cacheManager.getCache(Constants.LINKED_AUTH_CACHE).get(linkTransactionId, OIDCTransaction.class);	//NOSONAR getCache() will not be returning null here.
     }
+
+    @CacheEvict(value = Constants.CONSENT_CACHE, key = "#transactionId", condition = "#transactionId != null")
+    public OIDCTransaction getWebConsentedTransaction(String transactionId) {
```

Comment:
can we rename the method without web, since that is not followed as a convention in this layer.